modules/easyddb_dams_media_browser/easyddb_dams_media_browser.features.field_instance.inc

```diff
@@ -118,10 +122,10 @@ function easyddb_dams_media_browser_field_default_field_instances() {
     'description' => '',
     'display' => array(
       'default' => array(
-        'label' => 'hidden',
+        'label' => 'above',
```

Comment:
Is this supposed to be so?

Reply:
It doesn't really matter since the field is hidden